### PR TITLE
New Feature: Set default model

### PR DESCRIPTION
**Pull request by e-lisa**
**Proposal:** When using `openfacesee-gd` as a regular user, loading your model(s) every single time you start the application is  tedious. To improve the quality of life for users we should consider adding a button (or other UI element) that allows the user to save the current model as their "default model". This change will save the the `app_config` and persist through application starts.

**Summary of Changes:**

- Added a new button to `MainView.gd` labeled `Set as default`
  - If the model is already saved as default, the button will be disabled
- Added new app_config entry to store default model path: `["settings"]["default_model"] 
- Added public member `AppManager.set_model_default()` to enable setting and saving the default model
- Added public member `AppManager.get_current_model_path()` to get the current active model (can be different then default)
- Added public member `AppManager.is_current_model_default()` to enable checking if the current active model is the default (used for making "Set as default" button enabled/disabled
- Added `export/.gitkeep` so app_config exists by default (enabling saving app_config)

Please let me know if you have any feedback or require test coverage for this feature.

Thanks & Keep up the great work as always!

### CODE REVIEW

File: screens/gui/model-view/ModelView.gd [hunk 1]
```diff
@@ -106,7 +115,25 @@ func _setup_right(config: Dictionary) -> void:
...
 			elif c is InputLabel:
 				initial_properties[c.name] = c.get_value()
-	
+
+	if not right_container.outer.get_node_or_null("set_model_default_button"):
+		var set_model_default_button: Button = Button.new()
+		set_model_default_button.name = "set_model_default_button"
+		set_model_default_button.text = "Set as default"
+		set_model_default_button.size_flags_vertical = SIZE_EXPAND_FILL
+		set_model_default_button.size_flags_vertical = SIZE_EXPAND_FILL
```
Comment (you-win) [inline screens/gui/model-view/ModelView.gd:124]: lines 123 and 124 are repeated

**Review by you-win — CHANGES_REQUESTED** [review]

**Comment by you-win** [comment]
### tldr: the config file is due for extensive changes in the future

One item to note that's slightly related to this PR is that the config file is probably going to change significantly. At least with my usage, using presets is mostly broken. 

The design I had in my head was something like:
```
root (app-config.json)
- metadata
  - some global key-value pairs
  - anything stored in the last unfinished tab (App Settings)
- presets
  - <preset_name>
    - model
    - pose
    - features
  - ... more presets ...
```

Right now the file is messy and prone to bleeding data over to other models.

Nothing you need to change, just notifying you that your `app-config.json` data will be rolled into the `metadata` field in the future.
